### ISSUE TITLE
ORCID is not rendering in the word document

### ISSUE THREAD

**Issue by wolass** [benmarwick/rrtools#156]
**Describe the bug**
The prvided templates support the use of orcid, but it does not being rendered in the actual paper. So the orcid stays only in the yaml header

Describe the steps to reproduce the behavior:
1. create a new conpendium using the `rrtools::use_compendium()` function
2. call `rrtools::use_analysis()`
3. render quatro document
4. See lack od ORCID in the final docx output.

**Expected behavior**
Optimally we should get an ORCID icon next to the researchers name with a link to a respective ORCID profile when clicked on.

**Comment by benmarwick** [comment]
Yes, you're right, I think this has only ever worked when the output is HTML. I haven't seen any R Markdown or Quarto template that accomplishes this for docx output. Have you seen any?

It might require combining this https://github.com/pandoc/lua-filters/tree/master/author-info-blocks with https://stackoverflow.com/questions/77882406/how-to-access-meta-data-from-quarto-yaml-in-lua-script
